.eslintrc

```diff
@@ -61,6 +61,7 @@
       "objectsInObjects": false,
       "arraysInObjects": false
     }],
+    "object-shorthand": [2, "properties"],
```

Comment:
This rule will make the linter complain every time we use a non-shorthand syntax. Shall I make the lint level `error` or `warning`? Do I have to separate this change to another PR?